tests/components/__snapshots__/grid.test.ts.snap

```diff
@@ -120,7 +120,7 @@ height: 1px;
 exports[`error tooltip can display error tooltip 1`] = `
 <div
   class="o-popover"
-  style="z-index: 19; display: block; max-height: 824px; max-width: 697px; top: 187px; left: 336px;"
+  style="z-index: 19; display: block; max-height: 824px; max-width: 697px; top: 87px; left: 136px;"
```

Comment:
Could you investigate why the position of the tooltip moved in the snapshot ? Your PR should really impact that right ?

Reply:
Sure, I'll investigate further to confirm the cause.

Reply:
Hi @hokolomopo, Upon investigation, I couldn't determine the exact cause of the tooltip's position change, but it seems that using `createFigure` may be a contributing factor. If you have an idea about the way creating a figure affects tooltip positioning, could you please guide me how to proceed further? I'm less familiar with this aspect of snapshots to be frank.

Reply:
I investigated a bit, and the problem was that you were importing `createFigure` from the `figure.test.ts` file. But by loading this file, you call a `mockGetBoundingClientRect` which changes the coordinates of stuff. I changed the figure to a chart and we should be good :)

Reply:
Ohh, Great, thank you for your investigation and solution!